### PR TITLE
change control plane namespace

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Towards: https://github.com/giantswarm/giantswarm/issues/13371

Changing the name used for control plane prometheus to reflect the installation name.

Tested on `godsmack`

```
$ k get ns godsmack-prometheus
NAME                  STATUS   AGE
godsmack-prometheus   Active   46m
$ k get ns kubernetes-prometheus
Error from server (NotFound): namespaces "kubernetes-prometheus" not found
```

image

### CODE REVIEW

File: service/controller/control-plane/resource.go [hunk 1]
```diff
@@ -161,5 +161,39 @@ func newResources(config resourcesConfig) ([]resource.Interface, error) {
...
 
-	return resources, nil
+	{
+		resources, err = ControlPlaneWrap(resources, config)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	// TODO: remove this block in the next release.
```
Comment (JosephSalisbury) [inline service/controller/control-plane/resource.go:171]: See https://github.com/orgs/giantswarm/projects/153#card-46018679 for tracking

**Review by JosephSalisbury — APPROVED** [review]
lgtmiiwfy

**Comment by JosephSalisbury** [comment]
@TheoBrigitte why the picture of the alertmanager target, though?

**Review by omegas27 — APPROVED** [review]

**Comment by TheoBrigitte** [comment]
> @TheoBrigitte why the picture of the alertmanager target, though?

I got confused, this was already there before.

**Pull request merged**